components/alert/alert.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap");
+body {
+  font-family: "Source Sans Pro", sans-serif;
+  font-family: "Ubuntu", sans-serif;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 1rem;
+  width: 440px;
+  border: 1px solid grey;
+}
+.frame-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 1rem;
+}
+.alert {
+  padding: 1rem;
+  border-radius: 5px;
+}
+.alert-primary {
```

Comment:
Please keep all the colors as css variable

Reply:
Okay!